listen.py

```diff
@@ -43,6 +43,41 @@ def from_json(cls, j):
             data=j.get('track_metadata')
         )
 
+    @classmethod
+    def from_influx(cls, row):
+        """ Factory to make Listen objects from an influx row
+        """
+        dt = datetime.strptime(row['time'] , "%Y-%m-%dT%H:%M:%SZ")
+        t = int(dt.strftime('%s'))
+        mbids = []
+        if row.get('artist_mbids'):
```

Comment:
@alastair This is how I found the bug, initially I didn't have any listens with artist_mbid in the influx db. Everything worked fine, but as soon as I submitted a listen with artist_mbid, I started getting internal server errors.

It seemed to me that initially when there were no listens with mbids, the row returned by influx did not contain the field even though we always specify `''` (I think this is the equivalent of null in influx) to be put in there if there's no value. But as soon as a single listen with mbids got added, influx started returning `None` for the field for the listens with no mbids.